src/OrbitGl/FrameTrackOnlineProcessor.cpp

```diff
@@ -44,6 +44,7 @@ void FrameTrackOnlineProcessor::ProcessTimer(const orbit_client_protos::TimerInf
     // TID is meaningless for this timer (start and end can be on two different threads).
     constexpr const int32_t kUnusedThreadId = -1;
     frame_timer.set_thread_id(kUnusedThreadId);
+    frame_timer.set_function_id(function_id);
```

Comment:
I guess an issue here is (my bad!) that this code is in two places and was only updated in one. Should we change this and avoid the duplication? (Not in this PR, of course.)

Reply:
Agree completely! :)